invenio_app_ils/internal_locations/jsonresolvers/internal_location.py

```diff
@@ -28,12 +28,18 @@ def jsonresolver_loader(url_map):
     @get_pid_or_default(default_value=dict())
     def location_resolver(internal_loc_pid):
         """Return the Location record for the given Internal Loc. or raise."""
-        location_pid = get_field_value(InternalLocation, internal_loc_pid,
-                                       "location_pid")
+        location_pid = get_field_value(
+            InternalLocation, internal_loc_pid, "location_pid"
+        )
         location = Location.get_record_by_pid(location_pid)
-        del location["$schema"]
 
-        return location
+        return pick(
+            location,
+            "name",
+            "opening_exceptions",
+            "opening_weekdats",
```

Comment:
Watch out, spelling issue here!
```suggestion
            "opening_weekdays",
```